configurations/get-message.yml

```diff
@@ -0,0 +1,27 @@
+- step: get_message
+  type: get
+  args:
+    url: https://example.com/endpoint
+    query:
+      some_val: "Hello World"
+      another_val: 123
+  result: the_message
+
+- step: post_message
+  type: post
+  args:
+    url: https://example.com/endpoint
+    body:
+      some_val: "Hello World"
+      another_val: 123
+  result: the_message
+
+- step: assign
+  type: assign
+  assign:
+    stringValue: "Bürokratt"
+    integerValue: 2021
+
+- step: return_value
+  type: return
```

Comment:
Is this necessary?

Reply:
we need to use some value to determine which type of step it is, we can also use "call" like with post and get steps